src/Routine/Engine/Converter/NullableConverter.cs

```diff
@@ -11,11 +11,9 @@ protected override List<IType> GetTargetTypes(IType type)
 			if (!type.IsValueType) { return new List<IType>(); }
 			if (type.IsVoid) { return new List<IType>(); }
 			if (type.IsGenericType) { return new List<IType>(); }
-			if (!(type is TypeInfo)) { return new List<IType>(); }
+			if (type is not TypeInfo typeInfo) { return new List<IType>(); }
```

Comment:
"is not" iyimiş

Reply:
evet fena değil, zor oluyordu `is` ifadesini `not`lamak, toplamış, ama biraz da garip 🤷‍♂️